CBF_Git_Practice.md

```diff
@@ -0,0 +1,39 @@
+# NOTES ON WHAT I LEARNT ON GIT 
+Git is a version control system. It is used mainly for collaboration and keeping track of code changes.
+GitHub is a code hosting platform for version control and collaboration. It lets you and others work together on projects from anywhere
+Repository: a place where we store our files. 
+Remote:It can be a folder on cloud or on some websites like GitHub, GitLab or Bitbucket or anywhere. It serve as a backup if your machine was crashed by power fluctuations or any other problems. 
+
+### Commands in git :octocat:
+- Git –version: This is used to check the version of git
+	$ git –version
```

Comment:
It would be great if you could format your command examples, e.g. `$ git -version`. Check out the [Markdown formatting docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code) to see how

Reply:
ok. I will definitely do that now